config/environments/development.rb

```diff
@@ -6,6 +6,9 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
+  # using 3001 to avoid conflict with our other applications
```

Comment:
I am wondering if it would reduce our cognitive overhead if we settled for keeping our local dev env on port `3000` for Klaxon.

The only other app I can think of that uses port `3000` is the docs admin, which we won't have to develop on much in the near future. This way we don't have this extra variable of 

"are all our configurations set up for port 3000?" 

muddling our development progress

Reply:
I actually am _very_ excited about this suggestion, so I've made a [post-launch ticket for it](https://arcpublishing.atlassian.net/browse/NAPPS-1477?atlOrigin=eyJpIjoiMDRiZWQyZTZmZDU4NGQ5MGEzNDU0NTE1MzBhM2NjZWMiLCJwIjoiaiJ9). For the time being, I _think_ it's kind of important for development to not have these conflicting ports because caching (I learned from working on this) includes redirects — and both this project and Documents include redirects. And I'm not sure what else it could affect, but I would absolutely like to treat this as a temporary difference from Klaxon that we roll back once Documents is more off our plate.